e2e/vm/support_bundle_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package vm
+
+import (
+	"archive/zip"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+)
+
+var testSupportBundle = func(o *option.Option) {
+	ginkgo.Describe("Support bundles", func() {
```

Comment:
Is it possible to add couple of lines to validate the zip name and the unzipped folder name here?